cmd/simulator/load/loader.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/ava-labs/subnet-evm/cmd/simulator/config"
+	"github.com/ava-labs/subnet-evm/cmd/simulator/key"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func CreateLoader(ctx context.Context, config config.Config) (*WorkerGroup, error) {
+	// Construct the arguments for the load simulator
+	switch {
```

Comment:
I don't know if this matters, but wouldn't it make more sense to have:
```go
switch endpointCount := len(config.Endpoints) {
case 0:
  // ...
case endpointCount < config.Workers:
  // ...
}
```
IMO, function calls within the case arms or whatever they're called make things harder to read. I'm reaching here though, it's not that bad.

Reply:
I think `len(...)` is fine here. If it were another function call, I'd agree.